include/tvm/runtime/graph_runtime.h

```diff
@@ -26,25 +26,27 @@
 #define TVM_RUNTIME_GRAPH_GRAPH_RUNTIME_H_
 
 #include <dlpack/dlpack.h>
-#include <dmlc/json.h>
 #include <dmlc/memory_io.h>
+#include <dmlc/json.h>
 #include <tvm/runtime/ndarray.h>
 #include <tvm/runtime/packed_func.h>
-
+#include <tvm/runtime/graph_runtime_factory.h>
```

Comment:
We should still keep the graph runtime internal, as we only expose the results as a Module API

Reply:
I understand now why `graph_runtime.h` in the current folder now after your explanation. How about `vm.h`? I assume  `graph_runtime_factory.h` will support `vm` too in the future.

Reply:
It has to do with how would the user with the code. While some of the vm data structures are publically available(e.g. Closure), most of the vm should also become private. 

Putting it internal means that `graph_runtime_factory` is visible to all the components inside tvm, it will be visible to vm as well. For the users, they can always use `factory["resnet18"]` to create the model and `factory["get_params"]()` to obtain the parameters.

Reply:
Got it, thanks for explanation.